<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PyXlinkViewer</class>
 <widget class="QWidget" name="PyXlinkViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>774</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PyXlinkViewer</string>
  </property>
  <widget class="QPushButton" name="button_close">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>390</y>
     <width>101</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="button_open_xlink_file">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>131</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Open Xlink file</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="table_xlinks">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>80</y>
     <width>511</width>
     <height>291</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>4</number>
   </attribute>
   <column>
    <property name="text">
     <string>Chain 1</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Residue 1</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Chain 2</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Residue 2</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>CA distance</string>
    </property>
   </column>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>30</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="button_satisfied_colour">
      <property name="text">
       <string>Satisfied</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_satisfied_colour">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>30</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="button_violated_colour">
      <property name="text">
       <string>Violated</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_violated_colour">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="button_export">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>390</y>
     <width>101</width>
     <height>32</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>141</width>
     <height>126</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Show:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="check_satisfied">
      <property name="text">
       <string>Satisfied</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="check_violated">
      <property name="text">
       <string>Violated</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="check_inter">
      <property name="text">
       <string>Inter-chain</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="check_intra">
      <property name="text">
       <string>Intra-chain</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="check_mono">
      <property name="text">
       <string>Mono-links</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>165</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Threshold (Ã…):</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="doublespin_threshold">
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="value">
       <double>27.000000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Select object:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="list_select_object"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_6">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>380</y>
     <width>153</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>XLink width:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="doublespin_width">
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
      <property name="value">
       <double>0.500000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>30</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="button_mono_colour">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Mono</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_mono_colour">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>No. satisfied:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>No. violated:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_edit_satisfied">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>230</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_edit_violated">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>260</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_7">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>380</y>
     <width>154</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Mono size:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="doublespin_mono_size">
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
      <property name="value">
       <double>2.000000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
